package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "sigmadao-ui",
+	"name": "sigma-dao",
```

Comment:
```suggestion
	"name": "SigmaDAO",
```

Reply:
There are certain provision regarding naming convention, using this throwed me the error, will see if there is any solution to it

Reply:
Pinged you a better approach